conans/client/conf/config_installer.py

```diff
@@ -247,3 +279,35 @@ def configuration_install(app, uri, verify_ssl, config_type=None,
         else:
             configs = [(c if c != config else config) for c in configs]
         _save_configs(configs_file, configs)
+    try:
```

Comment:
I have realized that we already have a file for the "config install". Lets use the time of that file as the last time. No need to create further files!

Reply:
which file do you mean, `conan.conf`?